portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,38 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> greetings = new ArrayList<>(Arrays.asList("Bonjour!", "Hello!", "Ni hao!"));
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    String greetingsJSON = convertToJson(greetings);
+    response.setContentType("application/json;");
+    response.getWriter().println(greetingsJSON);
+  }
+
+  /**
+   * Converts a DataServlet instance into a JSON string using the Gson library. Note: We first added
+   * the Gson library dependency to pom.xml.
+   */
+  private String convertToJson(List<String> data) {
+    Gson gson = new Gson();
+    String json = gson.toJson(data);
+    return json;
```

Comment:
Can be inline like:

return new Gson().toJson(data);

Reply:
Done.